CHANGELOG.md

```diff
@@ -28,6 +28,12 @@ agent processed a particular event.
 - Added `pipelines` field to `Check` and `CheckConfig`.
 - Added `sensu_go_agentd_event_bytes` & `sensu_go_store_event_bytes` summary
 metrics to the `/metrics` endpoint.
+- Added the platform metrics log. This log contains a listing of core Sensu
+metrics in influx-line format. It is enabled by default but can be disabled
+with the --disable-platform-metrics flag. By default the log is appended to
+every 60s, and written to /var/lib/sensu/sensu-backend/stats.log.
```

Comment:
Why `/var/lib` and not `/var/log`?

Reply:
Good point! It should probably be /var/log.